INSTALL.md

```diff
@@ -57,8 +57,27 @@ Shogun is part of [homebrew-science](https://github.com/Homebrew/homebrew-scienc
     $ sudo brew install shogun
 
 ###Windows <a name="windows"></a>
-Shogun natively compiles under Windows using MSVC, see the [CI build](https://ci.appveyor.com/project/vigsterkr/shogun). We currently do not support a binary
-installer. If you are interested in packaging, documenting, or contributing otherwise, please contact us.
+We currently do not support a Shogun binary on Windows platform. Shogun natively compiles under Windows using MSVC. It is recommended to use " Visual Studio 14 2015 " or " MSBuild " to build Shogun which both have good support to C++11. 
+
+Run CMD as administrator with the following commands：
+```
+git clone https://github.com/shogun-toolbox/shogun.git C:\projects\shogun 
+git submodule -q update --init
+```
+Move to the source tree root and create the build directory.
+```
+cd C:\projects\shogun
+md build && cd build
+```
+You need [CMake](https://cmake.org/download/) to build Shogun. If you need Shogun examples in build, please see [Examples](#manual-examples).
+```
+cmake -G"Visual Studio 14 2015 Win64" -DCMAKE_BUILD_TYPE=Release -DBUILD_META_EXAMPLES=OFF -DENABLE_TESTING=ON ..
+```
+Compile
+```
+msbuild "C:\projects\shogun\build\shogun.sln" /verbosity:minimal /t:Clean /p:Configuration=Release /p:Platform=x64
```

Comment:
Yeah I hope you dont mind I took your changes, I was doing a general edit of the files and thought I might just include it.
Thanks a lot for that, it will be helpful for people